### PR TITLE
fix rendering after freeze

### PR DESCRIPTION
**Pull request by yegvla**
Sometimes when I run the hello world application, I will get a random image instead of the "Hello, World!" text, because the program exits with CODE_FREEZEFRAME, before vmes can render anything.

This pr will make vmes continue to render the buffer, even if the program exited with CODE_FREEZEFRAME.
When the program exited with CODE_FREEZEFRAME it can't do anything anyway because it already exited.

### CODE REVIEW

**Comment by emilmair** [comment]
I had encountered the same problem, that's why i moved the condition which sets 'freeze' to true AFTER rendering for a first time. While this fixed it for me, the app thread might take longer to start on your machine and therefore write to the SDL2 buffer after rendering and before vmes checks for the freeze exit code, resulting in random garbage on the screen.

TL;DR: Approved
